docbook/Admin_Guide/en-US/config/misc.xml

```diff
@@ -287,5 +287,12 @@
 				</warning>
 			</listitem>
 		</varlistentry>
+			<varlistentry>
```

Comment:
Confuses me, what do you want to add in rge Admin Guide?
$p_active_only is not a configuration option.